sdk/js/src/utils/consts.ts

```diff
@@ -27,7 +27,6 @@ export const CHAINS = {
   gnosis: 25,
   pythnet: 26,
   xpla: 28,
-  ropsten: 10001,
```

Comment:
I originally thought about leaving this as a reserved chain id to prevent it from being reused, but now I don't think it matters much, because this is a testnet chain